lib/components/narrative/line-itin/place-row.js

```diff
@@ -59,6 +65,10 @@ export default class PlaceRow extends Component {
     }
 
     const interline = leg && leg.interlineWithPreviousLeg
+    // TODO: This changeVehicles condition might should be showing the full stop
+    // place name because it could be helpful to give the user the stop viewer to
+    // click on in order to see what time the next vehicle will be arriving to the
```

Comment:
This comment could use a few grammatical fixes.

Reply:
I'm just going to go ahead and implement this `TODO`.